src/main/java/org/broadinstitute/dsde/consent/ontology/OntologyApp.java

```diff
@@ -28,6 +28,11 @@
 public class OntologyApp extends Application<OntologyConfiguration> {
 
     public static void main(String[] args) throws Exception {
+        String dsn = System.getProperties().getProperty("sentry.dsn");
+        if (null != dsn && !dsn.isEmpty()) {
+            RavenBootstrap.bootstrap(System.getProperties().getProperty("sentry.dsn"));
+            Thread.currentThread().setUncaughtExceptionHandler(UncaughtExceptionHandlers.systemExit());
```

Comment:
do you need to set the system-exit exception handler *before* the bootstrap?

Reply:
According to the plugin authors, Raven bootstrap happens first: https://github.com/tradier/dropwizard-raven
The idea is that we want to send anything that causes a startup error to sentry, and if Raven were not instantiated, nothing would be sent.

Reply:
ahhhh. I see. I thought you were trying to exit on any errors during the Raven bootstrap. All good.